Files/UserControls/Selection/ISelectedItems.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Files.UserControls.Selection
+{
+    public interface ISelectedItems
```

Comment:
Is this interface necessary? It's identical to `IList`

Reply:
@gave92 - I know it looks crazy :smile:

I didn't touch C# for a while, but I wanted to apply the same logic to both `Collections.IList` and `Collections.Generics.IList<object>`.

The reason for this is that I wanted to reuse the same strategy for both list and grid, but their respective Selected items have different types.

Reply:
Suggestion: `uiElement.SelectedItems.Cast<object>().ToList()` will convert an `IList` to an `IList<object>`
The reverse should be possible as well

Reply:
@gave92  - I cannot do it as I need a side-effect on the original collection

Reply:
And the reverse? var generic = (System.Collections.IList)uiElement.SelectedItems;

Reply:
Crashes the COM object :smile: